unstructured/partition/common.py

```diff
@@ -569,6 +569,14 @@ def document_to_element_list(
                 element.metadata.text_as_html = (
                     layout_element.text_as_html if hasattr(layout_element, "text_as_html") else None
                 )
+                try:
+                    if (isinstance(element, Title) and not element.metadata.category_depth) and any(
```

Comment:
and not element.metadata.category_depth will return false for 0 or None, is this intentional or should this be `and element.metadata.category_depth is None`

Reply:
fixed